website/addons/badges/templates/badges_user_settings.mako

```diff
@@ -96,6 +96,7 @@ You have not created any Badges.
 
 //todo fetch url from server
   var jsonToList = function(badge, id) {
+      // TODO: Rewrite this substantially before this feature is re-activated
```

Comment:
Per policy, reply to this one has been added to JIRA ticket.
https://openscience.atlassian.net/browse/OSF-4426?focusedCommentId=31238&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-31238